server/routerlicious/packages/services/src/mongodb.ts

```diff
@@ -422,6 +479,8 @@ export class MongoDb implements core.IDb {
 		private readonly retryEnabled = false,
 		private readonly telemetryEnabled = false,
 		private readonly mongoErrorRetryAnalyzer: MongoErrorRetryAnalyzer,
+		private readonly apiCounterIntervalMS: number,
+		private readonly apiFailureRateTerminationThreshold: number,
```

Comment:
Where is the entry point for apiFailureRateTerminationThreshold? Do we need to change the value.yaml or fluid-configmap? Thanks.

Reply:
I defined an default value:
```
// 1 means 100%, using 2 just for safety for incorrect calculations and meaning this feature disabled
const DefaultApiFailureRateTerminationThreshold = 2;
```

And the value is read from a config:
<img width="497" alt="image" src="https://github.com/microsoft/FluidFramework/assets/3719246/7c2ef026-751f-45b8-9cae-b9046ac73a88">

If config is missing, it will use the default value
<img width="741" alt="image" src="https://github.com/microsoft/FluidFramework/assets/3719246/17e1eaf6-d5f2-48cc-aae2-9cecbc3e04b6">